Procfile

```diff
@@ -1 +1,2 @@
-web: babel-node bin/server.js
+web:            babel-node bin/server.js server
+expireSentry:   babel-node bin/server.js expire-sentry
```

Comment:
Does this end up being useful?  As far as I can tell, the scheduler requires an explicit command, rather than something in Procfile